.github/CODEOWNERS

```diff
@@ -7,11 +7,11 @@
 # Please put the CODEOWNERS file in .github
 
 # Root
-* @smartcontractkit/foundations @smartcontractkit/core @smartcontractkit/bix-framework @smartcontractkit/core-solidity
+* @smartcontractkit/foundations @smartcontractkit/core @smartcontractkit/bix-framework
```

Comment:
Removed as core-solidity only owns Solidity related folder, not the top level which is largely Go based files.